packages/framer-motion/src/gestures/drag/__tests__/index.test.tsx

```diff
@@ -146,7 +147,7 @@ describe("dragging", () => {
         pointer.end() // + 3 = 14
         await onDragEnd.promise
 
-        expect(count).toBeGreaterThanOrEqual(12)
```

Comment:
I've taken a look at why this is happening and the feature is getting unmounted a frame earlier than before. This could be explained by before it was getting cleaned up by the returned Effect in a child component whereas now we clean up features on unmounting via the `ref` prop.

But the thing we're testing for is working as expected and I don't expect this to have any practical ramifications.